build.gradle

```diff
@@ -22,7 +22,9 @@ plugins {
 }
 
 dependencies {
+    compileOnly "com.enonic.xp:portal-api:${xpVersion}"
     compileOnly "com.enonic.xp:script-api:${xpVersion}"
+    compileOnly "com.enonic.xp:core-macro:${xpVersion}"
```

Comment:
This is worthless. This bundle doesn't export anything

Reply:
done